staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -264,6 +264,15 @@ const (
 	// NamesAccepted means the names chosen for this CustomResourceDefinition do not conflict with others in
 	// the group and are therefore accepted.
 	NamesAccepted CustomResourceDefinitionConditionType = "NamesAccepted"
+	// NonStructuralSchema means that the CustomResourceDefinition has one or more schemas which are not structural.
```

Comment:
clarify a NonStructuralSchema condition with value=True means this

Reply:
done